sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/test/TestBoundedTable.java

```diff
@@ -92,8 +95,19 @@ public TestBoundedTable addRows(Object... args) {
 
   @Override
   public PCollection<Row> buildIOReader(PBegin begin) {
+    PTransform<PBegin, PCollection<Row>> valueTransform;
+
+    if (rows.isEmpty()) {
+      valueTransform =
+          Create.empty(
+              SchemaCoder.of(
+                  schema, SerializableFunctions.identity(), SerializableFunctions.identity()));
+    } else {
+      valueTransform = Create.of(rows);
```

Comment:
if rows is empty, Create.of() will throw exception and ask for SchemaCoder

Reply:
why not just always set the schema, it will then handle both cases:

`Create.of(rows).withSchema(SchemaCoder.of(...))`